cityscape/tweet.py

```diff
@@ -19,6 +24,7 @@ def search(self, term, count=100, since_id=None):
 
         tweets = tweepy.Cursor(api.search, q= term, since_id=since_id, count=100, result_type="recent").items(count)
 
+>>>>>>> 223db0a2fcbac668667608e9bbc031b69e6ae54f
```

Comment:
There is an un-merged conflict in this `tweet.py`. Looks like you have another merge conflict, so you should pull master as well.